### ISSUE TITLE
dont use star exports

### ISSUE THREAD

**Issue by airtonix** [estruyf/vscode-typescript-exportallmodules#14]
Coming from a python background, star exports are a no no...

it's fine for small projects,

but for massive monorepos, this starts to chew into the editors discovery time...

```
🛑 bad

export * from './lol/sucks/to/be/you/this/is/my/entire/app';
```

```
✅ good
export { 
 All,
 The,
 Things,
 In,
 My,
 App,
 Are,
 Listed,
 Here,
 As,
 These,
 Exports
} from './lol/sucks/to/be/you/this/is/my/entire/app';
```

**Comment by estruyf** [comment]
Thanks @airtonix for the suggestion. 🙏

**Comment by estruyf** [comment]
This feature has been implemented and is available in version 2.8.0 thanks to @mrheault
